tests/postgres/derives.rs

```diff
@@ -12,6 +12,13 @@ use std::ops::Bound;
 #[sqlx(transparent)]
 struct Transparent(i32);
 
+// Also possible for single-field named structs
+#[derive(PartialEq, Debug, sqlx::Type)]
+#[sqlx(transparent)]
+struct TransparentNamed {
+    field: i32,
+}
```

Comment:
It's not really enough to throw a derive invocation in this file, we do want to test that it actually works.

You've already written a functional test in a different file, so this is redundant.